### ISSUE TITLE
No name is provided for combo box: A11y_VS Code extension for API Center_Open API Documentation_Combobox_Name Role Value.

### ISSUE THREAD

**Issue by AnKushSingh05** [microsoft/vscode-azureapicenter#107]
### GitHub Tags
#A11yTCS; #A11ySev2; #Mac; #WCAG4.1.2; #GH_VSCodeextensionforAPICenter_Mac_Apr2024; #DesktopApp; #VS Code Extension for API Center; #Name Role Value; #AILimited; #A11yMAS; #device;

### Environment Details:
Application Name: VS Code Extension for API Center
OS: Windows 11 Enterprise 23H2 (OS build 22631.3296)

### Repro Steps:

1. Create an API center in azure portal.
2. Open Visual studio Code.
3. Tab till API center extension in the left side panel and press enter.
4. Tab till definition under the Api name which you have created, and press enter.
5. The name which you have provided to the definition will appear for ex-"openapi4", right click on it.
6. Go to "Open API Documentation" using the down arrow and press enter.
7. Demo conference API screen will appear.
8. Tab till combo box.
9. Verify whether the name is provided for combo box or not.

### Actual Result:
No name is provided for combo box.
**Same issue is also observed for the button available in the code.**

### Expected Result:
Name should be provided for combo box.

### User Impact:
Screen reader users will not get the information about the control if name is not provided.

### Attachment:

**Comment by AnKushSingh05** [comment]
@formulahendry Can you please provide us the standalone sample app or Code pen link to route this bug to external team.

**Comment by AnKushSingh05** [comment]
IRT Bug logged: https://dev.azure.com/CSS-IRT/CSS%20IRT/_workitems/edit/220345

**Comment by Yash14j** [comment]
### GitHub Tags:
#External;#External:SwaggerUI;#eDAD:3p:220345;ThirdParty;

**Comment by AnKushSingh05** [comment]
External bug: https://dev.azure.com/CSS-IRT/CSS%20IRT/_workitems/edit/220345

**Comment by AnKushSingh05** [comment]
Adding the tag #ResolvedByTT;
